mod/Jailbreak.Warden/Global/WardenBehavior.cs

```diff
@@ -88,15 +98,34 @@ public bool TrySetWarden(CCSPlayerController controller)
 		logs.Append( logs.Player(_warden), "is now the warden.");
 
 		_unblueTimer = _parent.AddTimer(3, UnmarkPrisonersBlue);
+        mute.PeaceMute(hadWarden ? MuteReason.WARDEN_TAKEN : MuteReason.INITIAL_WARDEN);
+
+        // Always store the stats of the warden b4 they became warden, in case we need to restore them later.
```

Comment:
Uh this comment is incorrect, we only store the stats of the guard who's about to claim warden IF there have been no previous wardens...